RecoLocalCalo/EcalRecProducers/plugins/EcalCPUUncalibRecHitProducer.cc

```diff
@@ -27,6 +27,7 @@ class EcalCPUUncalibRecHitProducer : public edm::stream::EDProducer<edm::Externa
   void produce(edm::Event&, edm::EventSetup const&) override;
 
 private:
+  bool isPhase1_;
```

Comment:
We mostly seem to have `isPhase2_` used in cmssw, which I understand is basically defined as `!isPhase1_`. In the interest of consistency, it might be useful to switch here.